src/all/kamyroll/src/eu/kanade/tachiyomi/animeextension/all/kamyroll/Consumyroll.kt

```diff
@@ -62,45 +67,40 @@ class Kamyroll : ConfigurableAnimeSource, AnimeHttpSource() {
 
     // ============================== Popular ===============================
 
-    override fun popularAnimeRequest(page: Int): Request =
-        GET("$baseUrl/content/v1/updated?channel_id=$channelId&limit=20")
+    override fun popularAnimeRequest(page: Int): Request {
+        val start = if (page != 1) "start${(page - 1) * 36}&" else ""
```

Comment:
You seem to have forgotten to add an `=` after `start`

Reply:
Haha it seems I haven't scrolled pass page 1 in my testing, thanks for catching that